### ISSUE TITLE
stop working after 9 nft upload

### ISSUE THREAD

**Issue by Terlain1** [nftdevs/NFTs-Upload-to-OpenSea#97]

**Comment by SeanFrohman** [comment]
Yeah it's doing a captcha, every 10 uploads now you have to prove you're not a robot. Which basically makes the software completely ineffective now. I was doing the 10 at a time thing up until today now it's not even clicking this sign button for my wallet to confirm the price. Looks like the software might need a little bit of an overhaul.
